UPGRADE-API-1.13.md

```diff
@@ -63,166 +129,122 @@
+    * `GET `/shop/taxon-translations/{id}`
+    * `GET `/shop/product-translations/{id}`
+    * `GET `/shop/product-variant-translations/{id}`
+    * `GET `/shop/shipping-method-translations/{id}`
 
-    `Sylius\Bundle\ApiBundle\CommandHandler\Account\SendAccountRegistrationEmailHandler`:
-    ```diff
-        public function __construct(
-            private UserRepositoryInterface $shopUserRepository,
-            private ChannelRepositoryInterface $channelRepository,
-    -       private SenderInterface $emailSender,
-    +       private Sylius\Bundle\CoreBundle\Mailer\AccountRegistrationEmailManagerInterface $accountRegistrationEmailManager,
-        ) {
-        }
-    ```
+   The fields those endpoint were exposing are available on their respective translation subject resources.
 
-    `Sylius\Bundle\ApiBundle\CommandHandler\Account\SendAccountVerificationEmailHandler`:
-    ```diff
-        public function __construct(
-            private UserRepositoryInterface $shopUserRepository,
-            private ChannelRepositoryInterface $channelRepository,
-    -       private SenderInterface $emailSender,
-    +       private Sylius\Bundle\CoreBundle\Mailer\AccountVerificationEmailManagerInterface $accountVerificationEmailManager,
-        ) {
-        }
-    ```
+1. Update in Translations Handling
 
-    `Sylius\Bundle\ApiBundle\CommandHandler\Account\SendResetPasswordEmailHandler`:
-    ```diff
-        public function __construct(
-    -       private SenderInterface $emailSender,
-            private UserRepositoryInterface $shopUserRepository,
-            private ChannelRepositoryInterface $channelRepository,
-    +       private Sylius\Bundle\CoreBundle\Mailer\ResetPasswordEmailManagerInterface $resetPasswordEmailManager,
-        ) {
-        }
-    ```
+   The process for creating or updating translations via the API has been refined. Now, the locale for each translation
+   is determined directly from its key, making the explicit transmission of the `locale` field redundant. Although the
+   API continues to support the explicit sending of the `locale` field, it is essential that this explicitly sent locale
+   matches the key in the translation array. In cases of a mismatch between the key and an explicitly sent locale, the
+   API will respond with a `Sylius\Bundle\ApiBundle\Exception\TranslationLocaleMismatchException`.
 
-    `Sylius\Bundle\ApiBundle\CommandHandler\Checkout\SendOrderConfirmationHandler`:
-    ```diff
-        public function __construct(
-    -       private SenderInterface $emailSender,
-            private OrderRepositoryInterface $orderRepository,
-    +       private Sylius\Bundle\CoreBundle\Mailer\OrderEmailManagerInterface $orderEmailManager,
-        ) {
-        }
-    ```
+1. Disabled product and taxon editing at `/admin/product-taxons/{id}` operation to improve data integrity. To modify a
+   productTaxon, remove the existing association and create a new one.
```

Comment:
```suggestion
1. Product and taxon editing at `/admin/product-taxons/{id}` operation has been disabled to improve data integrity. To modify a product-taxon relationship remove the existing one and create a new one.
```
We still allow modifying the `position` field, so the sentence `To modify a
   productTaxon (...)` might be misleading.